stdlib/Random/src/Random.jl

```diff
@@ -326,6 +326,8 @@ When only one argument is passed besides the optional `rng` and is a `Tuple`, it
 as a collection of values (`S`) and not as `dims`.
 
 
+See also [`rand!`](@ref) to act in-place, and [`randn!`](@ref) for normally distributed numbers.
```

Comment:
Since it's the docstring for `rand`, linking to `randn` seems more appropriate as the version of this function for normally distributed numbers. Perhaps:

```suggestion
See also [`randn`](@ref) for normally distributed numbers, and [`rand!`](@ref) and [`randn!`](@ref) for the in-place equivalents.
```

Reply:
To me it looks like you just reordered the two parts of the sentence